api/src/test/java/uk/gov/dhsc/htbhf/eligibility/controller/EligibilityControllerTest.java

```diff
@@ -42,4 +52,60 @@ void shouldGetEligibility() {
         assertThat(response.getBody()).isNotNull();
         assertThat(response.getBody()).isEqualTo(eligibilityResponse);
     }
+
+    @Test
+    void shouldReturnBadRequestForMissingNino() {
+        PersonDTO person = aPersonWithNoNino();
+
+        var benefit = restTemplate.postForEntity(ENDPOINT_URL, person, EligibilityResponse.class);
+
+        Assertions.assertThat(benefit.getStatusCode()).isEqualTo(BAD_REQUEST);
```

Comment:
You don't need the `Assertions.` on all your `assertThat` method calls in this test class.

Reply:
Removed.